### PR TITLE
style: refactor build leading function style, width, and padding parameters for custom node leading builder

### PR DESCRIPTION
**Pull request by EchoEllet**
## Description

*Move the style logic from a conditional expression into a function call for improved readability.*

The refactoring simplifies the code by replacing nested ternary operators with if statements, improving the readability a bit.

## Related Issues

- *Related #2146*

## Type of Change

- [ ] ✨ **New feature:** Adds new functionality without breaking existing features.
- [ ] 🛠️ **Bug fix:** Resolves an issue without altering current behavior.
- [x] 🧹 **Code refactor:** Code restructuring that does not affect behavior.
- [ ] ❌ **Breaking change:** Alters existing functionality and requires updates.
- [ ] 🧪 **Tests:** Adds new tests or modifies existing tests.
- [ ] 📝 **Documentation:** Updates or additions to documentation.
- [x] 🗑️ **Chore:** Routine tasks, or maintenance.
- [ ] ✅ **Build configuration change:** Changes to build or deploy processes.

### CODE REVIEW

**Review by CatHood0 — APPROVED** [review]
I hadn't thought of this way! Thanks for the change. I think this refactor is more than excellent, now it's much easier for anyone to read.

We could use this PR to do the same with other parameters that have this one.

File: lib/src/editor/widgets/text/text_block.dart [hunk 1]
```diff
@@ -269,30 +269,35 @@ class EditableTextBlock extends StatelessWidget {
...
+          );
+        }
+        if (isCheck) {
+          return null;
+        }
+        return defaultStyles.code!.style.copyWith(
+          color: defaultStyles.code!.style.color!.withOpacity(0.4),
+        );
+      }(),
       width: isOrdered || isCodeBlock
```
Comment (CatHood0) [inline lib/src/editor/widgets/text/text_block.dart:301]: As I said above, we could do the same with width which can be improved

Comment (EchoEllet) [reply lib/src/editor/widgets/text/text_block.dart:301]: I will update the PR soon.

> As i said before

I'm not quite sure where you said it or missed the notification.

Comment (EchoEllet) [reply lib/src/editor/widgets/text/text_block.dart:301]: > As i said before,

I'm not quite sure where you said it or missed the notification.

Will do the same with `width` and `padding` soon.

**Review by singerdmx — APPROVED** [review]

**Review by CatHood0 — APPROVED** [review]
Nice! Thanks for taking the time to do these changes.

**Pull request merged**